Core/Core/CourseSync/CourseSyncProgress/ViewModel/CourseSyncProgressInfoViewModel.swift

```diff
@@ -48,25 +48,27 @@ class CourseSyncProgressInfoViewModel: ObservableObject {
 
         interactor.observeDownloadProgress()
             .receive(on: scheduler)
+            .throttle(for: .milliseconds(300), scheduler: scheduler, latest: true)
```

Comment:
I know it's not the first time this is introduced, but what's the reason behind the 300 ms? why 300?

Reply:
I removed both throttles.